src/providers/solana.ts

```diff
@@ -120,17 +120,6 @@ export default class SolanaProvider extends BloctoProvider implements SolanaProv
   }
 
   async connect(): Promise<void> {
-    const existedSDK = (window as any).solana;
-    if (existedSDK && existedSDK.isBlocto) {
-      return new Promise((resolve) => {
-        existedSDK.once('connect', (pubkey: PublicKey) => {
-          this.accounts = [pubkey.toBase58()];
-          resolve();
-        });
-        existedSDK.connect();
-      });
-    }
-
```

Comment:
Is this the correct behavior we want? Cuz by design, we should use the native SDK (iOS or Android) when Web SDK is being called in the in-app browser; and this is what this piece of code tries to achieve here.

Reply:
https://github.com/portto/blocto-sdk/blob/448a0990364771df001caf5b5d9df24e58f62372/src/providers/solana.ts#L122-L133

change use on function and generate accounts use native SDK publicKey